core/src/env_platform/env_service.bal

```diff
@@ -117,7 +118,7 @@ service envservice on ep0 {
         path: "/maps/validate-map",
         body: "body"
     }
-    resource function validateCoordinates(http:Caller caller, http:Request req, Location body) returns error? {
+    resource function validateTheLocation(http:Caller caller, http:Request req, Location[] body) returns error? {
```

Comment:
Here it should be `Area` instead of `Location` right?

Reply:
Oh just saw this, yeah it should be right?

Reply:
@yashodgayashan I'm confused cause we have both `Area` and `Location` can you explain the use of these two to us?

Reply:
Location is a point where it has latitude and longitude. Area is the set of locations which will cover an area.

Reply:
So does that mean the location includes the area?

Reply:
Area is a collection of Location objects. It's used to construct the polygon. Correct me if I am wrong @yashodgayashan

Reply:
I didn't get it. Simply I mean Area is an array of locations.

Reply:
> Area is a collection of Location objects. It's used to construct the polygon. Correct me if I am wrong @yashodgayashan

yes it is.